docs/code_contribution_guidelines.md

```diff
@@ -485,8 +485,31 @@ statements and select statements.
 ```
 
 If one is forced to wrap lines of function arguments that exceed the 80
-character limit, then a new line should be inserted before the first stanza in
-the comment body.
+character limit, then identation must be kept on the following lines.
+
+**WRONG**
+```go
+func foo(a, b, c,
+) (d, error) {
+
+func bar(a, b, c) (
+  d, error,
+) {
+```
+**RIGHT**
+```go
+func foo(a, b,
+  c) (d, error) {
+
+func bar(a, b, c) (
```

Comment:
I personally don't think we should use this style since the open parenthesis at the end of the line tricks the eye into believing the definition is finished even though it continues on the next line.